spyder/plugins/editor/plugin.py

```diff
@@ -954,29 +954,70 @@ def get_plugin_actions(self):
         self.recent_file_menu = QMenu(_("Open &recent"), self)
         self.recent_file_menu.aboutToShow.connect(self.update_recent_file_menu)
 
-        file_menu_actions = [
+        from spyder.plugins.mainmenu.api import (
+            ApplicationMenus, FileMenuSections)
+        # New Section
+        self.main.mainmenu.add_item_to_application_menu(
             self.new_action,
-            MENU_SEPARATOR,
+            menu_id=ApplicationMenus.File,
+            section=FileMenuSections.New,
+            before_section=FileMenuSections.Restart)
+        # Open section
+        open_actions = [
             self.open_action,
             self.open_last_closed_action,
             self.recent_file_menu,
-            MENU_SEPARATOR,
-            MENU_SEPARATOR,
+            ]

```

Comment:
Please unindent this bracket four spaces so it's at the same level of `open_actions`.